gatsby-node.js

```diff
@@ -9,6 +9,8 @@ const axios = require('axios')
 
 const BASE_URL = 'http://localhost:5000/fetch?url='
 const HomeTemplate = path.resolve('./src/templates/HomePage.tsx')
+const AuthorTemplate = path.resolve('./src/templates/Author.tsx')
+const ArticleTemplate = path.resolve('./src/templates/Article.tsx')
 
 // TODO: add action from create HomePage
 const createHomePage = async (createPage) => {
```

Comment:
would prefer removing the parenthesis here, since async createPage => {...} is also valid but less cluttered to look at